internal/http/services/owncloud/ocdav/get.go

```diff
@@ -131,7 +131,16 @@ func (s *svc) handleGet(ctx context.Context, w http.ResponseWriter, r *http.Requ
 	}
 	defer httpRes.Body.Close()
 
-	copyHeader(w.Header(), httpRes.Header)
+	// copy only the headers relevant for the content served by the datagateway
```

Comment:
can u add a check if the headers exist and only set the value if it exists?

Reply:
done.